src/physical/datatypes/storage_value.rs

```diff
@@ -88,6 +52,27 @@ impl StorageValueT {
     }
 }
 
+macro_rules! storage_value_try_into {
+    ($variant:ident => $dst:ty) => {
+        impl TryFrom<StorageValueT> for $dst {
+            type Error = ();
```

Comment:
I think this is okay for now, we might want to make it a variant of `crate::Error::Error` at some point (possibly via an intermediate `StorageValueTryIntoError`, which could be a unit newtype).